x-pack/plugins/cases/public/components/edit_connector/index.tsx

```diff
@@ -248,13 +249,20 @@ export const EditConnector = React.memo(
             </Form>
           </DisappearingFlexItem>
           <EuiFlexItem data-test-subj="edit-connector-fields-form-flex-item">
-            {(currentConnector == null || currentConnector?.id === 'none') && // Connector is none or not defined.
-              !(currentConnector === null && selectedConnector !== 'none') && // Connector has not been deleted.
-              !editConnector && (
-                <EuiText size="s">
+            {!editConnector && permissionsError ? (
```

Comment:
@cnasikas can you check my logic here?

Reply:
I think it is possible that a user with write privileges for cases could encounter a permissions error (they don't have actions and connectors permissions). In this situation the edit icon will be displayed and the user could attempt to edit the connectors.

Reply:
We could make an argument that it doesn't make sense to be able to edit the connectors if there was a permissions error though 🤔